database_actions.py

```diff
@@ -363,3 +366,13 @@ def phase_shift(database, roomId):
             Message('Voting Phase', 'The night is over! Who is the mafia?'))
         return True
     return False
+
+
+def handle_disconnect(database, roomId, userId):
+    room_data = get_room(database, roomId)
+    print('disconnect')
+    if room_data.status == 'pre-game':
+        room_data.players[:] = [
+            x for x in room_data.players if x.userId != userId]
```

Comment:
you don't need the `[:]`
```suggestion
        room_data.players = [
            x for x in room_data.players if x.userId != userId]
```
```py
a = [1,2,3]
a[:] = [x for x in a if x > 2]
print(a) # 3
a = [1,2,3]
a = [x for x in a if x > 2]
print(a) # 3
```

Reply:
oh that function is just totally unused now im just gonna get rid of it entirely haha